resolwe_bio/processes/expression/salmon.py

```diff
@@ -356,6 +358,9 @@ def run(self, inputs, outputs):
         # Zip the gene-level abundance estimates
         (Cmd["gzip"]["-c", reads_name] > reads_name + output_suffix)()
 
+        # Zip the gene-level count estimates
+        (Cmd["gzip"]["-c", counts] > counts + ".gz")()
```

Comment:
Personally, I would create a new variable `counts_gz = counts + ".gz"` and use that here and when assigning it to outputs.